### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#31448]
#### VFS team name
Authenticated Experience

#### VFS product name
Notification Preferences

#### Point of Contact/Reviewers
Josh Kim (Accessibility)

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
On mobile, "Profile menu" is a `h1` which creates two `h1` elements on the page. On desktop, "Profile" is a `h4`, which skips several heading levels. 

### Link, screenshot or steps to recreate
1. Log in to staging using test user 202
2. Reduce the screen width to a mobile view
3. Use headingsMap or your screen reader to confirm that "Profile menu" is a `h1`
4. Increase the screen width to a desktop view
5. Use headingsMap or your screen reader to confirm that "Profile" is a `h4`

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

---

### Platform Recommendation
On both mobile and desktop, since `profile` is a navigation's heading that comes before the `h1`, it should be a `h2` following [WCAG guidance H42](https://www.w3.org/TR/WCAG20-TECHS/H42.html). Utility classes can be used to reduce the size of the `h2`.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by MarciMcGuireGCIO** [comment]
Note - may assign to Erik for Sprint 61

**Comment by tmitchellgcio** [comment]
@joshkimux Ready for validation

**Comment by joshkimux** [comment]
@tmitchellgcio Brilliant! ⭐ 
Just saw this in the other ticket. Confirming here as well that it is now consistent on both mobile and desktop 👍

**Comment by MarciMcGuireGCIO** [comment]
Great work, @tmitchellgcio ! @joshkimux thank you as always for your expertise! 😃
